manifests/client.pp

```diff
@@ -97,10 +97,12 @@
     notify  => Service[$agent_service_name],
   }
 
+  Concat::Fragment {
+    target => 'ossec.conf',
+    notify => Service[$agent_service_name]
+  }
+
```

Comment:
If I understood the docs correctly the defaults for a resource type are valid only in the same scope (i.e.: same manifest). This should not cause issues.
Case in point: on the same server there is another `concat::fragment` managing puppet configuration and the above does not interfere (unless it's a coincidence).